utils/protobuf-schema-utilities/src/test/proto/TestOrderingSyntax2References.proto

```diff
@@ -0,0 +1,69 @@
+package io.apicurio.registry.utils.protobuf.schema.syntax2.references;
```

Comment:
Can you also add an example proto for syntax 3 reference?

Reply:
There is a reference case in `proto3`

Reply:
The test proto seems much simpler than this. Does internal/external also work for proto3?

Reply:
Added detailed test cases now. Thanks